plza-fe/src/components/users/reviews/UserReview.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Icon, Item, Label, Rating } from "semantic-ui-react";
+import { userDeleteReview } from "../../../redux/actions/userActions.js"
+import ReviewUpdate from "./ReviewUpdate.js";
+
+const UserReview = ({ review }) => {
+  const user = useSelector((state) => state.user);
+  const [toggleEdit, setToggleEdit] = useState(false);
+  const dispatch = useDispatch();
+  const [reviewToEdit, setReviewToEdit] = useState({});
+
+  const editReview = (review) => {
+    setReviewToEdit(review);
+  };
+
+  console.log(user);
```

Comment:
Would it be possible to get this console.log removed?

Reply:
Done. https://github.com/Lambda-School-Labs/lets-get-pizza-fe/pull/62/commits/1ecc211a05654ae0e2914f87ca6183a3060b2ad9